metadata.yaml

```diff
@@ -26,8 +26,8 @@ peers:
     interface: upgrade
 
 provides:
-  zookeeper:
-    interface: zookeeper
+  database:
+    interface: zookeeper_client
```

Comment:
How should we introduce this change?
The way mongodb did is to add an `obsolete` relation with the old interface. Is this what we would like to do?

Reply:
Resolved: we keep one interface backward compatible